doc/articles/migrating-apps.md

```diff
@@ -2,11 +2,11 @@
 uid: Uno.Development.MigratingApps
 ---
 
-# Migrating a UWP-only application codebase to an Uno Platform application
+# Migrating a WinUI/UWP-only application codebase to an Uno Platform application
```

Comment:
@jeromelaban, @morning4coffe-dev  seems to me that maybe the migration section would need a bit more adjustments, especially for WinUI -> Uno App
![image](https://github.com/user-attachments/assets/cf3da41b-5999-4592-a72e-3d963ff401ee)

cf. https://discord.com/channels/1182775715242967050/1182775715729526908/1370524131631562753

![image](https://github.com/user-attachments/assets/6388de7e-2ba7-4053-bfb1-c99f559a65a3)

Reply:
Let's do that one in another PR